TaskWebApp/Web.config

```diff
@@ -103,6 +103,10 @@
         <assemblyIdentity name="System.Runtime.CompilerServices.Unsafe" publicKeyToken="b03f5f7f11d50a3a" culture="neutral" />
         <bindingRedirect oldVersion="0.0.0.0-5.0.0.0" newVersion="5.0.0.0" />
       </dependentAssembly>
+      <dependentAssembly>
```

Comment:
looks like this dependency was added just this time. do we need this?

Reply:
No we dont need it. I will remove it.